src/Text/Pandoc/PDF.hs

```diff
@@ -70,8 +70,8 @@ makePDF :: String              -- ^ pdf creator (pdflatex, lualatex, xelatex,
         -> WriterOptions       -- ^ options
         -> Pandoc              -- ^ document
         -> PandocIO (Either ByteString ByteString)
-makePDF program pdfargs writer opts doc = do
-  case takeBaseName program of
+makePDF program pdfargs writer opts doc
+  = case takeBaseName program of
```

Comment:
agreed that the `do` can be removed, but the `=` should stay on the upper line...